engine/configurator/conf_yaml.c

```diff
@@ -370,364 +551,586 @@ parse_config_yaml_dependency_entry(yaml_document_t *d,
  * @return Status code.
  */
 static te_errno
-parse_config_yaml_dependency(yaml_document_t            *d,
-                             yaml_node_t                *n,
-                             cs_yaml_target_context_t   *c)
+parse_config_yaml_dependency(yaml_document_t *d, yaml_node_t *n,
+                             object_type *obj)
 {
-    cytc_dep_entry *dep_entry;
     te_errno        rc;
 
-    if (n->type == YAML_SCALAR_NODE)
+    if (n->type == YAML_SCALAR_NODE)/* Is it really used? */
```

Comment:
Missing space before comment.

Reply:
I've added a space. But I really want to know if if it used somewhere. I've never seen such examples. If they do exist then I should remove the comment, should I? In there are no such examples maybe I should remove this part of the code, should I?